GroupMeClientApi/GroupMeClient.cs

```diff
@@ -46,6 +53,11 @@ public GroupMeClient(string authToken)
         /// </summary>
         internal RestClient ApiClient { get; } = new RestClient(GroupMeAPIUrl);
 
+        /// <summary>
```

Comment:
Having a single ApiClient for the entire client instance would be preferable. The root URL in the client can be overridden by the RestRequest, so we shouldn't need multiple for different root API endpoints. See [Image Attachment](https://github.com/alexdillon/GroupMeClientApi/blob/master/GroupMeClientApi/Models/Attachments/ImageAttachment.cs#L48). My suggestion here would be to get rid of ApiV4Client, and remove GroupMeAPIUrl from the constructor for ApiClient.

Reply:
To extend of this, CreateRestRequest() should be changed to CreateRestRequestV3, and should include the GroupMeAPIUrlV3 base url. A new one, CreateRestRequestV4 would be created in the same style. And finally, a new CreateRawRestRequest() is probably needed that contains no base URL. Existing calls to CreateRestRequest would have to be checked. Anything that includes /Group, or /Chat, or similar would just be changed to the V3 version and require no further changes. This will make these calling methods much clearer as to which version of the Api endpoint they are actually invoking. For calls to CreateRestRequest where a non-V3 endpoint or a full URL is passed in, CreateRawRestRequest would be the suitable replacement.  

As a side thought, the auth key might be able to be made private now? Not sure on that completely though.